mongodbatlas/resource_mongodbatlas_search_index.go

```diff
@@ -485,7 +485,7 @@ func unmarshalSearchIndexAnalyzersFields(mappingString string) []admin.ApiAtlasF
 	}
 	var fields []admin.ApiAtlasFTSAnalyzers
 	if err := json.Unmarshal([]byte(mappingString), &fields); err != nil {
-		log.Printf("[ERROR] cannot unmarshal search index mapping fields: %v", err)
+		log.Printf("[ERROR] can not unmarshal search index mapping fields: %v", err)
```

Comment:
how important is this change? should we just leave it like that? because eventually I don't think this makes it "better english" 😄

Reply:
this was to test file change detection worked fine :-) i'll revert it

Reply:
here I changed a file, for config I added PR label [run-testacc-generic](https://github.com/mongodb/terraform-provider-mongodbatlas/labels/run-testacc-generic) so i test different ways to trigger the tests